lib/rules/no-mocha-arrows.js

```diff
@@ -16,21 +15,31 @@ module.exports = {
     create(context) {
         const sourceCode = context.getSourceCode();
 
+        // eslint-disable-next-line max-statements
```

Comment:
Not sure how you feel about this rule, but I didn't see an obvious way to break up this method without passing multiple return values.

Reply:
It’s not a hard requirement. I don’t have a better approach in mind for now. As long as we have a good test coverage we can always refactor the code later 😉.